### DISCUSSION TITLE
Reduce Basijis By Offering Support

### DISCUSSION THREAD

**Discussion by homeinametronome** [WeAreMahsaAmini/Core#486] [Ideas]
Support for Defecting Basijis 

Many Basijis don’t like their jobs but they may feel as though they can’t quit. There must be support to address every single hesitation they may have so they can resign as soon as possible. 

The more resignations, the faster the revolution is established.

There needs to be an easy and persuasive source of information for them. It could be a hotline or a website. 

Loss of income? Here are ways you can retrain for a new job. 

Feel like it’s too late and you are on the blacklist? No worries, all is forgiven. 

Loss of friends? We are here and we won’t judge.

Do you need someone to talk to without judgement? We are here.

Do you want to learn English? Someone is here for you to practice with. 

We will need a team of experts, especially war experts, psychological experts, because we are dealing with barbaric people and we need to convince them to put down their weapons the second we have established a connection with them. There needs to be sophisticated verification processes to make sure they are truly defecting. This sort of thing has been done before. Ukraine offers support for Russian soldiers who defect. It happens in all wars. I can support this project with ideas, graphic design, I know basic html/WordPress, I can only provide text in English. My Farsi literacy skills are basic and rusty.

**Comment by redflake** [comment]
That's a wonderful Idea! Thank you!
I had the same discussion today with my friend.
If you imagine the last moment of the revolution, that is exactly the time when the soldiers turn to the people. Why?
Because of profit, it could be money or some psychological reward as you mentioned.
So let's create that exact moment

I wait for the others to get the same vision, like or comment, then we can start
I would help by programming, and I know many to help for the Farsi language, mine is not perfect too
